src/main.rs

```diff
@@ -1694,10 +1694,8 @@ mod tests {
     impl FileKind {
         fn content(self) -> &'static [u8] {
             match self {
-                FileKind::Good => {
-                    b"    \n#Hello, world!\
-                    \n#Hello, world! Autogenerated      "
-                }
+                FileKind::Good => b"    \n#Hello, world!\
+                    \n#Hello, world! Autogenerated      ",
```

Comment:
I think maybe this is the result of a pre-1.40 formatter? Maybe an auto-formatter run by your editor?

Reply:
Yeah, I was on 1.38.0. 6 week release cycle got me behind